tests/test_simulator.py

```diff
@@ -0,0 +1,50 @@
+from unittest import TestCase
+
+from unittest.mock import Mock, patch
+
+from roulette import Simulator, Game, Martingale, Table, Wheel
+
+
+class TestSimulator(TestCase):
+    def setUp(self):
+        self.table = Table()
+        self.martingale = Martingale(self.table)
+        self.wheel = Wheel()
+        self.game = Game(self.wheel, self.table)
+        self.simulator = Simulator(self.game, self.martingale)
+
+    def test_gather_appends_max_stake_and_rounds_played_after_one_session(self):
```

Comment:
whenever planning to write test name with **and**, think whether the test should be broken down in multiple tests. Single test should test single behavior.

Reply:
test should focus on behavior not implementation details. A better name could be simulator_gathers_max_stake avoiding word append, which is a function name

Reply:
Done.